otx/mpa/modules/datasets/task_adapt_dataset.py

```diff
@@ -10,8 +10,16 @@
 
 
 @DATASETS.register_module()
-class TaskAdaptEvalDataset(object):
-    """Dataset wrapper for task-adative evaluation."""
+class TaskAdaptEvalDataset:
+    """Dataset wrapper for task-adaptive evaluation.
+
+    Attributes:
+        model_classes:
+        **kwargs:
+
+    Notes:
+        TODO[Eugene/Jaeguk]: check if this class is still valid in OTX
```

Comment:
It's not required for tiling, however, it's still being used.
 https://github.com/openvinotoolkit/training_extensions/blob/67105dcdd302699f2d39ebca6831561ba6f16692/otx/mpa/det/stage.py#L184

Since it's going to be used anyway according to Harim and Jaeguk. It's better including it in data pipeline. Therefore there will be a PR doing this as noted with TODO.

Reply:
@JihwanEom As Eugene said, it is still used in detection.